docs/release-notes/1.8.0.md

```diff
@@ -1,3 +1,4 @@
 ## New Features
 
 - Update the Istio version to 1.22.1 [#887](https://github.com/kyma-project/istio/pull/887). Read [Istio 1.22.0 Release Announcement](https://istio.io/latest/news/releases/1.22.x/announcing-1.22/), [1.22.0 Change Notes](https://istio.io/latest/news/releases/1.22.x/announcing-1.22/change-notes/), [1.22.0 Upgrade Notes](https://istio.io/latest/news/releases/1.22.x/announcing-1.22/upgrade-notes/), and [Istio 1.22.1 Release Announcement](https://istio.io/latest/news/releases/1.22.x/announcing-1.22.1/).
+- Update the Istio version to 1.22.2 [#912](https://github.com/kyma-project/istio/pull/912). Read [Istio 1.22.2 Release Announcement](https://istio.io/latest/news/releases/1.22.x/announcing-1.22.2/).
```

Comment:
shouldn't we keep just one version? in the end module version should upgrade to a single version.

Reply:
Moved to 1.8.1.